pkgs/by-name/sd/SDL2/package.nix

```diff
@@ -189,19 +189,18 @@ stdenv.mkDerivation (finalAttrs: {
     + "\n" +
     ''moveToOutput bin/sdl2-config "$dev"'';
 
-  # SDL is weird in that instead of just dynamically linking with
```

Comment:
I wouldn't mind if the big comment block was not changed multiple times...

Reply:
Solved.
But ugly.